### ISSUE TITLE
Update loss function

### ISSUE THREAD

**Issue by MokkeMeguru** [MokkeMeguru/TFGENZOO#3]
ref https://github.com/y0ast/Glow-PyTorch/issues/9

**Comment by MokkeMeguru** [comment]
I want to get same loss transition

**Comment by MokkeMeguru** [comment]
implement bits per dims
